python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/mistral/generate.py

```diff
@@ -0,0 +1,65 @@
+#
```

Comment:
what's the difference vs. the LLaMA2 example?

Reply:
> what's the difference vs. the LLaMA2 example?

The difference lies on prompt format and tokenizer class. Should we provide dedicated folder for each model, or just use one python script and list verified AWQ models in readme?

Reply:
Maybe one python script for CPU, one for GPU

Reply:
> Maybe one python script for CPU, one for GPU

Have updated. As https://github.com/analytics-zoo/nano/issues/780#issuecomment-1824427117, the correctness is currently checked through observing the output text.